### PR TITLE
Fix Crash In MA When Autoscale Y Pressed With No Data

### LINKED ISSUE #30267: Crash in MA When Autoscale Y Pressed With No Data
### Expected behavior
Should have no effect

### Actual behavior
crashes

### Steps to reproduce the behavior
1. Open MA
2. Check AutoScale y on the plot widget

### Platforms affected
all

### PR DESCRIPTION
**Pull request by Matt-Cumber**
**Description of work.**
Disabled plot window until data is loaded

**To test:**
1. Open Muon Analysis
2. Plot window should be disabled which stops being able to check autoscale y which prevents any crashes
3. Check that the plot window is re-enabled when data loaded
4. And pressing clear all disables it again
5. Do the same for Frequency Domain Analysis

Fixes #30267 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
when you put data into the GUI for the first time, the y scale is always rubbish

**Comment by Matt-Cumber** [comment]
> when you put data into the GUI for the first time, the y scale is always rubbish

I have this problem on other branches as well not just this one

**Comment by AnthonyLim23** [comment]
Thats new. Default the plotting to have autoscale ticked

**Review by AnthonyLim23 — DISMISSED** [review]
It all works now :shipit:

File: scripts/Muon/GUI/Common/plot_widget/plot_widget_presenter.py [hunk 1]
```diff
@@ -359,6 +360,10 @@ def plot_all_selected_data(self, autoscale, hold_on):
         workspace_list, indices = self._model.get_workspace_list_and_indices_to_plot(self._view.is_raw_plot(),
                                                                                      self._view.get_plot_type())
 
+        if workspace_list:
+            self._view.setEnabled(True)
+        else:
+            self._view.setEnabled(False)
```
Comment (DavidFair) [inline scripts/Muon/GUI/Common/plot_widget/plot_widget_presenter.py:366]: Is this worth adding a test for (especially as we have the mocking ready anyway) so this doesn't regress in the future?

**Comment by AnthonyLim23** [comment]
@DavidFair the build failure is unrelated

**Review by DavidFair — APPROVED** [review]
Thanks lets :shipit:

**Comment by DavidFair** [comment]
Failure unrelated

**Pull request merged**